data/schema.graphql

```diff
@@ -1264,6 +1287,15 @@ enum BuyerOutcomeTypes {
 }
 
 type BuyersPremium {
+  # A globally unique ID.
+  __id: ID!
+
+  # A type-specific ID.
+  id: String!
+
+  # A type-specific Gravity Mongo Document ID.
+  _id: String!
+
```

Comment:
so many ids lol x_x

What are the types here? it's an id specific to a type of... category? :o